src/NuGet.Core/NuGet.Protocol/Providers/ServiceIndexResourceV3Provider.cs

```diff
@@ -5,6 +5,9 @@
 using System.Collections.Concurrent;
 using System.Globalization;
 using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Runtime.Serialization.Formatters;
```

Comment:
Remove `System.Net` and `System.Runtime.Serialization.Formatters`.  They're unused.

Reply:
fixed in next iteration